utils/wp-odm_related-options.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+  class Wp_odm_related_Options {
+
```

Comment:
hey @Huyeng this allows to be able to setup and run automated tests. If you see under **tests/** the get_option function is being **Mocked**, which means that we can then define the returning value of this file for each one of the tests...